standalone-metastore/metastore-common/src/main/protobuf/org/apache/hadoop/hive/metastore/hive_metastore.proto

```diff
@@ -1966,13 +1966,15 @@ message AbortTxnRequest {
   int64 txnid = 1;
   string repl_policy = 2;
   TxnType txn_type = 3;
+  int64 errorCode = 4;
```

Comment:
should be optional, for backward compatibility

Reply:
Set field as optional. Done.